test/unit/resources/package_test.rb

```diff
@@ -139,6 +139,7 @@
   # undefined
   it 'verify package handling on unsupported os' do
     resource = MockLoader.new(:undefined).load_resource('package', 'curl')
+    _(resource.info).must_be_empty
```

Comment:
The only downside with using `must_be_empty` here is that if `resource.info` ever had a regression introduced that returned an empty string, this would still pass.

It might be safer to say `.must_equal({})`

Reply:
Great catch! I have updated it to match what you suggested.